ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-main.yaml

```diff
@@ -46,6 +50,24 @@ test_binary_build_commands: |
   make build-openshift-ci-test-bin
   go mod vendor
 tests:
+- as: fast-forward
+  commands: |
+    git clone -b master https://github.com/openshift/release.git /tmp/release
+    repo-brancher \
+        --fast-forward \
+        --confirm \
+        --config-dir=/tmp/release/ci-operator/config/openshift/assisted-image-service/ \
+        --username=openshift-merge-robot \
+        --token-path=/secrets/github-credentials-openshift-merge-robot/oauth \
+        --current-promotion-namespace=ocp-kni \
+        --current-release=ocm-2.4 \
+        --future-release=ocm-2.4
+  container:
+    from: repo-brancher
+  cron: '@hourly'
+  secrets:
+  - mount_path: /secrets/github-credentials-openshift-merge-robot
```

Comment:
I suggest creating an account that you own and manage that you would use for this. Our robots are unfortunately not to be shared, they have too many other rights.

Reply:
Okay will start to ask around about getting a separate bot. The repos are in `openshift` organization so it will probably have to be added in the organization settings.